openforcefield/tests/test_molecule.py

```diff
@@ -56,9 +58,11 @@ def assert_molecule_is_equal(molecule1, molecule2, msg):
 
 def test_cheminformatics_toolkit_is_installed():
     """Ensure that at least one supported cheminformatics toolkit is installed."""
-    if RDKIT_UNAVAILABLE and OPENEYE_UNAVAILABLE:
+    # Can't just call all_subclasses here -- AmberToolsToolkitWrapper won't suffice for this test
+    if not(RDKitToolkitWrapper.toolkit_is_available) and not(OpenEyeToolkitWrapper.toolkit_is_available):
+
         msg = 'No supported cheminformatics toolkits are installed. Please install a supported toolkit:\n'
-        msg += str(SUPPORTED_TOOLKITS)
+        msg += 'OpeneEyeToolkitWrapper or RDKitToolkitWrapper'
```

Comment:
We really need the toolkits themselves to be installed, and want to communicate to users how to install them. Perhaps we should instead print `toolkit_wrapper.installation_instructions` instead, which provides info on how to install the toolkits themselves (rather than the wrapper)?